src/analyze/analyze-plot.c

```diff
@@ -168,7 +168,9 @@ static void plot_tooltip(const UnitTimes *ut) {
         assert(ut->name);
 
         svg("%s:\n", ut->name);
-
+        svg("Activating: %d.%.3d\n", (int)ut->activating / 1000000, (int)ut->activating % 1000000);
+        svg("Activated: %d.%.3d\n", (int)ut->activated / 1000000, (int)ut->activated % 1000000);
```

Comment:
Maybe better to use `FORMAT_TIMESPAN(ut->activating, USEC_PER_SEC)` ?

At least, please use `USEC_FMT` or `PRI_USEC` rather than `%d`, and drop the casts.
Also, please use `USEC_PER_SEC` rather than `1000000`.

Reply:
Used PRI_USEC and USEC_PER_SEC